webapp/actions/team_actions.jsx

```diff
@@ -75,3 +75,21 @@ export function removeUserFromTeam(teamId, userId, success, error) {
         }
     );
 }
+
+export function getTeamByName(teamName, success, error) {
+    Client.removeUserFromTeam(
```

Comment:
I think this is a mistake, right? Also don't need to add an action if it's not being used anywhere